modules/packer/custom-image/image.pkr.hcl

```diff
@@ -134,9 +134,9 @@ build {
   # provisioner "powershell" blocks
   dynamic "provisioner" {
     labels   = ["powershell"]
-    for_each = var.powershell_scripts
+    for_each = var.windows_startup_ps1
     content {
-      script = provisioner.value
+      inline = split("\n", provisioner.value)
```

Comment:
Should we handle `\n\r` as a "new line" since it's "friendly" module?

Reply:
I think the DOS newline is `\r\n` (reverse). Strictly speaking, the current code ought to handle it.

WDYT of filtering out all `\r` with empty string?

Reply:
Can splitting be avoided altogether? 
```hcl
inline = [provisioner.value]
```

Reply:
I'am ok with keeping `\n`, because I'm not sure if there is indeed a problem.